tech-committee-charter.md

```diff
@@ -61,6 +61,30 @@ The development process will include a process for the TC to oversee and approve
 
 The TC and entire technical community will follow any processes as may be specified by the Cloud Native Computing Foundation relating to the intake and license compliance review of contributions, including the CNCF IP Policy.
 
+### Sponsorship Requirements
+
+The TC will establish and maintain sponsorship across the SIGs and subprojects of OpenTelemetry.
+
+SIGs will be classified into three classifications of Sponsorship requirement:
+
+* *Escalation Sponsorship*: The TC sponsor will be a primary means of escalation of cross-project concerns from the SIG. This communication is primarily
+  offline, e.g. via CNCF slack or issue communication. The TC sponsor will check-in with SIG leadership quarterly.  Additionally, the TC will aide the SIG
+  in developing quarterly road maps during this check-in.
+* *Guiding Sponsorship*: The TC sponsor is an active participant of the SIG. The sponsor will help direct the SIGs efforts to match the technical goals of OpenTelemetry.
```

Comment:
Are guide responsibilities _in addition to_ escalation responsibilities, or would there be a separate TC member who is the escalation assignee? What if there are multiple TC members active in a SIG?

Reply:
This is stated below:

> Higher level sponsorship automatically fulfills lower level sponsorship requirements

Maybe that should be above the list already to avoid confusion, because I had the same question initially.